extension/content.js

```diff
@@ -328,6 +368,11 @@ document.addEventListener('DOMContentLoaded', () => {
 			if (isCommit()) {
 				addPatchDiffLinks();
 			}
+
+			if (isPRFiles()) {
```

Comment:
That's funny - I originally had the POC checking for both, but I had *thought* late last night that the sticky bar wasn't enabled on commit pages. I just double-checked, and I was wrong :)